package.json

```diff
@@ -0,0 +1,14 @@
+{
+	"name": "@wordpress/security-whitepaper",
+	"version": "1.0.0",
+	"description": "WordPress Security White Paper",
+	"private": true,
+	"scripts": {
```

Comment:
Could we define `engines`, or at least include an `.nvmrc` file? I don't think we need to match Core in this case. Using the latest version (currently `24.x`) is probably fine as well.

Also just curious why no lock file.